infrastructure/http/HTTPRequestExecutionAction.go

```diff
@@ -35,14 +33,15 @@ func CreateAction() HTTPAction {
 		MaxIdleConnsPerHost: 10,
 	}
 	var instance = HTTPAction{
-		client: &http.Client{Transport: tr},
+		client:    &http.Client{Transport: tr},
+		extractor: NewDefaultsExtractor(),
+		State:     HttpActionState{},
 	}
 	return instance
 }
 
 //Execute ...
 func (instance HTTPAction) Execute(ctx context.Context, executionContext core.ExecutionContext) core.ExecutionResult {
```

Comment:
Do we have any direct tests for this, or do we rely on higher level test coverage?